examples/diffeq.jl

```diff
@@ -0,0 +1,385 @@
+# # Integrating Agents.jl with DifferentialEquations.jl
+#
+# Leveraging other best-in-class packages from the Julia ecosystem is one of the many
+# strengths Agents.jl provides over alternative ABMs.
+#
+# The [DifferentialEquations.jl](https://github.com/SciML/DifferentialEquations.jl) package
+# [consistently outperforms](http://www.stochasticlifestyle.com/wp-content/uploads/2019/08/de_solver_software_comparsion.pdf)
```

Comment:
I don't think this fits here. It is not our place to discuss DE solver comparison, we only show how easy it is to use Agents with one of them.